### PR TITLE
Define resources as a seperate Docker stack

### PR DESCRIPTION
**Pull request by FoRVaiS**
### Summary
- Suffix all resources (PMA, DB) with `-resource`
- Create targets to spin up resources, all commands do the same thing
    - `make res`
    - `make resource`
    - `make resources`
- Move build/process prereqs to the environment targets

### Testing
1. Run `make res` (or any of the other commands listed above)
2. Run `docker ps` and you should see PMA and MySQL
image

### CODE REVIEW

File: apps/api/.env.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 API_HOST=0.0.0.0
 API_PORT=3000
 
-DATABASE_HOST=mysql-dev
+DATABASE_HOST=mysql-resource
```
Comment (tgrigoruk) [inline apps/api/.env.example:4]: With a change like this, what's the best way to notify everyone to update their local `.env` file?

File: makefile [hunk 2]
```diff
@@ -27,15 +27,22 @@ build:
...
 	$(MAKE) run SERVICE=dev
 
-prod:
+prod: build down
 	$(MAKE) run SERVICE=prod
 
+res: resources
+resource: build down
+resources: build down
+	$(MAKE) run SERVICE=resource
```
Comment (tgrigoruk) [inline makefile:43]: What is the difference in use between these, if any?

**Review by tgrigoruk — APPROVED** [review]

File: makefile (hunk 2 above)
Comment (FoRVaiS) [reply makefile:43]: It's just an alias, some people might not remember if it's `make resource` or `make resources`. It's just better to make both forms of the command run the same thing to reduce frustration caused by such a small problem.

File: apps/api/.env.example (hunk 1 above)
Comment (FoRVaiS) [reply apps/api/.env.example:4]: I'm really not sure, there's really no way to automate this process so I think the simplest way to just to @everyone and refrain from making these changes too often.

File: makefile (hunk 2 above)
Comment (tgrigoruk) [reply makefile:43]: I didn't realize at the time that line 43 runs after any of the previous 3 commands - it's obvious now that they're the same 😅

**Pull request merged**